airbyte-integrations/connectors/source-mailchimp/integration_tests/configured_catalog.json

```diff
@@ -69,6 +69,34 @@
       "cursor_field": ["send_time"],
       "primary_key": [["id"]],
       "destination_sync_mode": "append"
+    },
+    {
+      "stream": {
+        "name": "list_members",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh", "incremental"],
+        "source_defined_cursor": true,
+        "default_cursor_field": ["last_changed"],
+        "source_defined_primary_key": [["id"]]
+      },
+      "sync_mode": "incremental",
+      "cursor_field": ["last_changed"],
+      "primary_key": [["id"]],
+      "destination_sync_mode": "append"
+    },
+    {
+      "stream": {
+        "name": "get member info",
```

Comment:
The name of the stream here should match the defined name in streams.py

Reply:
Done :)